app/components/post-search-results-affiliate-item.js

```diff
@@ -3,6 +3,7 @@ import Component from '@ember/component';
 import { track, trackActions } from '../utils/track';
 
 export default Component.extend({
+  classNames: ['post-search-results-affiliate-item'],
```

Comment:
Interesting. Does that mean there is always a div for each component?

Reply:
Yeah, ember takes care of it

Reply:
Hmmm. I'm not sure if I like that or find it to be wasteful if you have the template.

Reply:
Regardless it will always* be there.